main.cpp

```diff
@@ -64,7 +64,12 @@ int main(int argc, char *argv[]) {
 		path = "./";
```

Comment:
Please add a quick summary of the purpose of this main and the process occurring inside.